cachito/workers/cleanup_job.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import logging
+from datetime import datetime, timedelta
+
+import requests
+
+from cachito.workers.requests import get_requests_session
+from cachito.workers.config import get_worker_config
+from cachito.errors import CachitoError
+
+log = logging.getLogger(__name__)
+
+auth_session = get_requests_session(auth=True)
+session = get_requests_session()
+config = get_worker_config()
+
+
+def main():
+    """
+    Get all completed requests from the Cachito API.
+
+    :raise CachitoError: if the request to the Cachito API fails
+    """
+    # Get all completed requests
+    response = session.get(config.cachito_api_url + 'requests?state=complete')
```

Comment:
What if there's more than one page of completed requests that need to be marked as stale? This only grabs a single page.

Reply:
Nice catch! Done :)